webpack.common.js

```diff
@@ -1,55 +1,55 @@
 const path = require('path')
-const StyleLintPlugin = require('stylelint-webpack-plugin')
 const { VueLoaderPlugin } = require('vue-loader')
```

Comment:
This is the kind of apps for which a standard webpack config would works flawlessly too!
@ChristophWurst https://github.com/skjnldsv/vueexample/pull/238

Reply:
I just copy and pasted it from the app-tutorial ;)

Reply:
Yes :)
I mean, we discussed it last week on the js channel, about a nice vue webpack library which we can easily use in our apps and extend!
Christoph was doubting it, but we have tons of apps who have pretty much all the same webpack files and would benefit from just relying on an external template I think!

What do you think @georgehrke ?

Reply:
I'm generally in favour of everything that makes things easier for standard apps. It should work for most apps (mostly excluding those which rely on async imports for moment locales).

Reply:
Let's try to deploy this and integrate it in other places!